packages/adapter-d1/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@auth/d1-adapter",
+  "version": "0.1.2",
+  "description": "A Cloudflare D1 adapter for Auth.js",
+  "homepage": "https://authjs.dev",
+  "repository": "https://github.com/nextauthjs/next-auth",
+  "bugs": {
+    "url": "https://github.com/nextauthjs/next-auth/issues"
+  },
+  "files": [
+    "dist",
+    "README.md"
+  ],
+  "author": "Josh Schlesser <josh@schlesser.dev>",
+  "contributors": [],
+  "main": "dist/index.js",
```

Comment:
Okay so a few last things, can you convert this to ESM only? 

See the `package.json` from the `dynamodb` adapter (https://github.com/nextauthjs/next-auth/blob/main/packages/adapter-dynamodb/package.json).

Specifically the `type: module`, `types`, and `exports` map :pray:

Reply:
@ndom91 , thanks for shepherding me through this.  I think these are the right changes.